configs/taco/OR.yaml

```diff
@@ -27,3 +27,6 @@ links:
 - name: quaternary
   url: https://public.tableau.com/profile/oregon.health.authority.covid.19#!/vizhome/OregonCOVID-19Update/HospitalCapacity
 
+- name: senary
```

Comment:
Do we want to give it a more descriptive name than "senary"?

Reply:
should i rename it to weekly_pdf for now @julia326 ?

Reply:
i vote yes!

Reply:
We also don't have a quinary here, so that's a name option too